specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "description": "Custom model identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "resultId",
+            "in": "path",
+            "description": "Analyze operation result identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeOperationResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model": {
+            "$ref": "./examples/AnalyzeOperationResult.json"
+          }
+        }
+      }
+    },
+    "/custom/models/{modelId}/copy": {
+      "post": {
+        "summary": "Copy Model",
+        "description": "Copy model artifacts to target subscription.",
+        "operationId": "CopyCustomModelAsync",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "modelId",
+            "in": "path",
+            "description": "Model identifier to copy.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "copyRequest",
+            "in": "body",
+            "description": "Request object for copying model.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CopyRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request to copy model artifacts queued successfully.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "Result id to track the progress of copy operation. Result of copy operation can be obtained only when the operation status becomes \"succeeded\" in status field."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Copy Model": {
+            "$ref": "./examples/CopyBatch.json"
+          }
+        }
+      }
+    },
+    "/custom/models/{modelId}/copyResults/{resultId}": {
+      "get": {
```

Comment:
Do we need this API? Why not reuse GET /custom/models/{modeld} assume the copyResult POST request will return a new modeld?

Reply:
Are you suggesting this to be a sync API?

Reply:
I'm suggesting not having this API at all. Copy returns a model ID that will be created, then use GET /custom/models/{modelId} to track the new model.